client/src/cmdscript.c

```diff
@@ -424,6 +429,17 @@ static int CmdScriptRun(const char *Cmd) {
 
         // clean up
         PyConfig_Clear(&py_conf);
+#else
+        wchar_t *py_args[argc + 1];
+        py_args[0] = Py_DecodeLocale(filename, NULL);
```

Comment:
Using `Py_DecodeLocale()` will allocate memory which needs to be freed at the end.

Reply:
There is the free at line 426.
The code is the one from commit #38cb327991501325aab6b24310e5428dc0369ab0